### PR TITLE
[Cherrypicked] Redshift source and connector plugin added.

### PR DESCRIPTION
**Pull request by vikasrathee-cs**
Added Redshift Source plugin for 6.9.x versions.
Cherrypicked from https://github.com/data-integrations/database-plugins/pull/485

### CODE REVIEW

File: database-commons/src/main/java/io/cdap/plugin/util/DBUtils.java [hunk 1]
```diff
@@ -59,7 +59,7 @@
 public final class DBUtils {
   private static final Logger LOG = LoggerFactory.getLogger(DBUtils.class);
 
-  private static final Calendar PURE_GREGORIAN_CALENDAR = createPureGregorianCalender();
+  public static final Calendar PURE_GREGORIAN_CALENDAR = createPureGregorianCalender();
```
Comment (MrRahulSharma) [inline database-commons/src/main/java/io/cdap/plugin/util/DBUtils.java:62]: Why this change ?

Comment (vikasrathee-cs) [reply database-commons/src/main/java/io/cdap/plugin/util/DBUtils.java:62]: this was getting used in one of the class and was public in 1.11 release

Comment (itsankit-google) [reply database-commons/src/main/java/io/cdap/plugin/util/DBUtils.java:62]: If it is not getting used in any of the redshift class related to your change, let's revert. Otherwise LGTM.

**Review by itsankit-google — APPROVED** [review]

File: database-commons/src/main/java/io/cdap/plugin/util/DBUtils.java (hunk 1 above)
Comment (vikasrathee-cs) [reply database-commons/src/main/java/io/cdap/plugin/util/DBUtils.java:62]: It is getting used.
